memory-viz/src/types.ts

```diff
@@ -21,17 +21,30 @@ export const StylesSchema = z.union([
 export type Styles = z.infer<typeof StylesSchema>;
 
 export const DrawnEntitySchema = z.object({
-    name: z.string().optional(),
-    type: z.string().optional(),
-    x: z.number().optional(),
-    y: z.number().optional(),
-    id: z.union([z.number(), z.null()]).optional(),
+    name: z.string("Invalid input: valid inputs must be a string").optional(),
+    type: z
+        .string(
+            'Invalid input: valid inputs are "list", "set", "tuple", "dict", "int", "str", "None", "bool", "float", and "date"'
```

Comment:
MemoryViz actually supports an arbitrary string for type, so let's just use the same error message format as the others